docs/swagger.json

```diff
@@ -165,31 +300,45 @@
 					},
 					"404": {
 						"description": "Permit not found"
-					},
-					"405": {
-						"description": "Validation exception"
 					}
 				}
-			},
+			}
+		},
+		"/permits/applications/special-uses/commercial/temp-outfitters/{controlNumber}/files/{fileName}": {
 			"get": {
 				"tags": [
-					"Noncommercial Permit"
+					"Outfitter and Guide Permit"
 				],
-				"summary": "Find noncommercial permit by ID",
-				"description": "Returns a single permit",
-				"produces": [
+				"summary": "Get files of Outfitter and Guide Permit",
+				"description": "",
+				"consumes": [
 					"application/json"
 				],
+				"produces": [
+					"application/json",
+					"application/pdf",
+					"text/rtf",
+					"application/msword",
+					"application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+					"application/octet-stream"
```

Comment:
When user selects a particular content type MIME and has a validation error, it would not return the error in swagger. Selecting the application/json works for both successful and error cases.